core/backend/src/ECDb.ts

```diff
@@ -322,7 +322,7 @@ export class ECDb implements IDisposable {
    * See [ECSQL row format]($docs/learning/ECSQLRowFormat) for details about the format of the returned rows.
    * @internal
    */
-  public async queryRows(ecsql: string, bindings?: any[] | object, limit?: QueryLimit, quota?: QueryQuota, priority?: QueryPriority, restartToken?: string): Promise<QueryResponse> {
+  public async queryRows(ecsql: string, bindings?: any[] | object, limit?: QueryLimit, quota?: QueryQuota, priority?: QueryPriority, restartToken?: string, abbreviateBlobs?: boolean): Promise<QueryResponse> {
```

Comment:
6 optional arguments is unwieldy - did you consider bundling them up into an interface?

Reply:
looked at this - an interface would definitely be a more optimal solution but will break more things dependent on the backend. It's on the todo list for the future, but it's not in scope for this PR